pkg/k8shandler/factory/cronjob.go

```diff
@@ -0,0 +1,26 @@
+package factory
```

Comment:
this isn't really a factory in the normal paradigm, is it really necessary to break this out from the other `cronjob.go` file?